scalyr_agent/copying_manager/worker.py

```diff
@@ -489,6 +489,14 @@ def run(self):
                         # on the ground and advance.
                         if current_time - last_success > self.__config.max_retry_time:
                             if self.__pending_add_events_task is not None:
+                                if (
+                                    "parseResponseFailed"
+                                    in self.__pending_add_events_task.__receive_response_status
+                                ):
+                                    log.error(
+                                        "Repeatedly failed to parse response due to exception.  Dropping events",
```

Comment:
I thought it may not be a bad idea to also log partial response here, but I believe we will already log this here (https://github.com/scalyr/scalyr-agent-2/pull/863/files#diff-4cb9befeecb4697d270768d8487b47aa20810c540fa7a9b6c86800735e7927a2R704) so it's not needed to do it here again?

Reply:
Yes, I wanted to be explicit that the repeated failures were causing events to be dropped.  But I don't have a strong opinion & I'm ok with removing it if that seems better.

Reply:
Sorry, my comment was a bit confusing.

I do think that having an error log message here is a good idea. 

I just thought it may also be a good idea to include actual server response here as well (e.g. ``scalyr_util.remove_newlines_and_truncate(response, 1000)``), but I actually think it's not needed since it will already be logged in the client code under warning / info.